plugin_info.toml

```diff
@@ -11,7 +11,7 @@ license = 'MIT'
 
 [plugin-install]
 #packages required for your plugin:
-packages-required = ["pyvisa", "pymeasure", 'pymodaq>=4.0']
+packages-required = ["pyvisa", "pymeasure", 'pymodaq>=4.0', 'sqlalchemy', 'sqlalchemy_utils', 'psycopg2']
```

Comment:
why do you need sql stuff in this plugin?

Reply:
I used it when trying to save data in a SQL database, but indeed not relevant with the plugin